client/lib/transformer.dart

```diff
@@ -154,3 +162,5 @@ const _ANALYTICS_CODE = '''<script>
   ga('send', 'pageview');
 
 </script>''';
+
+const _SURVEY_CODE = '<script async="" defer="" src="//survey.g.doubleclick.net/async_survey?site=4vq7qmqpfn4zjduve64c4h6lqa"></script>';
```

Comment:
only api.dartlang.org should get this snippet. I don't think we want this actually in dartdoc-viewer. Probably more of a build-process thing when we build the SDK.